OrbitService/ProducerSideServiceImpl.cpp

```diff
@@ -241,9 +246,9 @@ void ProducerSideServiceImpl::SendCommandsThread(
                                                   },
                                                   &service_state_),
                                               kCheckExitSendCommandsThreadInterval);
-        service_state_mutex_.Unlock();
       } break;
     }
+    service_state_mutex_.Unlock();
```

Comment:
I wonder if this can be scoped somehow as well. Maybe check == case first with early continue?

Reply:
I don't understand the suggestion.